Also the CI builds fail because of the include:
```
In file included from /home/runner/work/vampyr/vampyr/src/vampyr/export_vampyr.cpp:24:
/home/runner/work/vampyr/vampyr/src/vampyr/trees/trees.h:6:10: fatal error: pybind11/Eigen.h: No such file or directory
    6 | #include <pybind11/Eigen.h>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
```
I really am not sure what version of pybind11 you're using locally: browsing the tags of pybind11 only `<pybind11/eigen.h>` exists in their repo (note the lowercase "e") Are you on a case-insensitive filesystem? https://pybind11.readthedocs.io/en/stable/advanced/cast/eigen.html

--- a/src/vampyr/trees/trees.h
+++ b/src/vampyr/trees/trees.h
@@ -3,7 +3,7 @@
 #include <filesystem>
 
 #include <pybind11/stl/filesystem.h>
-#include <pybind11/Eigen.h>
+#include <pybind11/eigen.h>
 
 #include <MRCPP/trees/FunctionNode.h>
 #include <MRCPP/trees/FunctionTree.h>

--- a/src/vampyr/trees/trees.h
+++ b/src/vampyr/trees/trees.h
@@ -62,6 +62,10 @@ template <int D> void trees(pybind11::module &m) {
         .def("quadrature",
              [](FunctionTree<D> *tree) {
 
+                if constexpr (D != 1) {
+                    throw std::runtime_error("quadrature only implemented for 1D");
+                }
+
                 // Current implementation only makes sense in 1D
 
                 std::vector<double> vec_pts;

